### DISCUSSION TITLE
Is it possible to add custom dashboards to the dashboards list?

### DISCUSSION THREAD

**Discussion by FsxShader2012** [teslamate-org/teslamate#1863] [Q&A]
Hey everyone,

I've created my own dashboard because I wanted to be able to see everything im interested in on one page.
I'm wondering if it is possible to add my newly created dashboard to the dashboards list?

This is the list I'm talking about:
<details>
<summary>Click to view Screenshot</summary>
<img width="185" alt="Bildschirmfoto 2021-07-25 um 19 08 52" src="https://user-images.githubusercontent.com/15424862/126907470-62686f16-af92-4c00-9bb6-85f393aec485.png">
</details>

Thanks for your help!

**Comment by adriankumpf** [comment]
This is only possible if you compile TeslaMate yourself. All dashboards (saved as JSON) stored in the `grafana/dashboards` directory are displayed in this list.
